src/pages/integrations/index.tsx

```diff
@@ -0,0 +1,279 @@
+import React from "react";
+import clsx from "clsx";
+import Layout from "@theme/Layout";
+import { useColorMode } from "@docusaurus/theme-common";
+import { Tag, TAGS, Integration, INTEGRATIONS } from "@site/src/integrations";
+
+import styles from "./index.module.css";
+
+export default function Integrations() {
+  const [categoryFilter, setCategoryFilter] = React.useState("");
+  const [searchQuery, setSearchQuery] = React.useState("");
+
+  let integrations = INTEGRATIONS.filter((i) => {
+    if (categoryFilter === "v1") {
+      return i.fcVersion === "v1";
+    }
+    return i.fcVersion === "v2";
+  });
+
+  integrations = integrations.filter((i) => {
+    const s = JSON.stringify(i).toLowerCase();
+    return (
+      s.includes(searchQuery.toLowerCase()) &&
+      s.includes(categoryFilter.toLowerCase())
+    );
+  });
+
+  const androidVisible = !!integrations.find((i) => i.slug === "android");
```

Comment:
Maybe wrap each of these in a `useMemo`? React isn't very smart and would re-compute this every time anything changes